### PR TITLE
Added exposer to MathJax.Config and callback after rendering

### PR DESCRIPTION
**Pull request by JayAndBee**
Hey, 

I have added some functionality to your directive: One can now specify the MathJax.Config({}) call in the .forRoot() method. Furthermore, a callback can be specified in an attribute to be executed once the rendering is done. Details are in the updated README.

Best,

JB

### CODE REVIEW

**Comment by JayAndBee** [comment]
Hi, can you take a look at the code. I noticed that you denied some other pull request where someone [https://github.com/davidshen84/ngx-mathjax/pull/6](url) tried to pass the mathjax config as well. Why don't you think it should be done this way?

**Comment by davidshen84** [comment]
Yes, I did not like how they use string concatenation to create a JS
function. I thought the new MathJax v3 would give some better way to
configure it. But, it seems they did not or I did not find the right way.

Meanwhile, it seems a lot of people need to customize MathJax.

**Comment by JayAndBee** [comment]
Well, I guess since you are already using an html-script tag to load MathJax into your global space, you may as well allow for the Mathjax.Hub.Config to be called via a string. Would you consider my pull request, please? 

Best.

**Comment by davidshen84** [comment]
Yes, please address those comments.

File: README.md [hunk 1]
```diff
@@ -43,7 +43,16 @@ import {MathJaxModule} from './src/app/math-jax/math-jax.module';
...
-      hostname: 'cdnjs.cloudflare.com'
+      hostname: 'cdnjs.cloudflare.com',
+      mathjaxconfigobject : {
+          'HTML-CSS': {
+            styles: {
+              '.MathJax_Display': {
+                'background-color': 'yellow',
+              },
+            },
+          },
```
Comment (JayAndBee) [inline README.md:54]: Test Comment

**Comment by JayAndBee** [comment]
I beg your pardon? Which comments do you mean? I can't see any comments in the "Files Changed".

**Comment by davidshen84** [comment]
I think you need to login to GitHub to see those comments. Anonymous
account won't see the comments.

**Comment by JayAndBee** [comment]
I am very sorry, I am new to github. If you mean specfic comments in the "Files Changed" tab, I cannot view those even though I am logged in. I haven't seen any other comments, either. Help would be appreciated.

**Comment by davidshen84** [comment]
Hi Jay,

If you want, I can transfer this repo to you. I really don't have time to
maintain this project. Do you want it?

**Comment by JayAndBee** [comment]
Sure, I can do that. Thank you.

**Comment by davidshen84** [comment]
You forked this repo. You need to rename or remove it first before I can transfer it to you.

**Comment by JayAndBee** [comment]
Hey,


I have renamed the ngx-mathjax forked repo. You can transfer it for me now.

Thank you for your work on the project.


Best,

**Comment by davidshen84** [comment]
Sorry, I am still getting the " JayAndBee already has a repository in the
davidshen84/ngx-mathjax network" error when trying to transfer the repo to
you.

**Pull request merged**